src/ati/remote/choose.py

```diff
@@ -0,0 +1,55 @@
+from ati.errors import InvalidRemoteError
+
+from ati.remote import s3_remote_state
+
+
+def _verify_s3(state : dict):
```

Comment:
Is the dict of verifies a reasonable way to do the function choice? 

by living in ati.remote, do you mean in the __init__, or another module?